ChangeLog

```diff
@@ -77,6 +77,9 @@ What's New in astroid 2.11.7?
 =============================
 Release date: TBA
 
+* Added support for ``usedforsecurity`` keyword to ``hashlib`` constructors.
+
+  Refs PyCQA/pylint#6017
```

Comment:
On second thought, I don't see the point of retesting this in pylint -- we don't need to bloat the `unexpected-keyword-arg` tests with a verification of every stdlib keyword.
```suggestion
  Closes PyCQA/pylint#6017
```